app/users/UserForm.js

```diff
@@ -0,0 +1,97 @@
+"use client";
+import { Formik } from "formik";
+import { Button } from "../components/shared/Button";
+import Link from "next/link";
+import {
+  initialValuesCreateUser,
+  submitHandler,
+  validationSchemaCreateUser,
+} from "./helper";
+import { createUser, editUser } from "../api/users";
+import TextField from "../components/shared/TextField";
+
+export default function UserForm({ user }) {
+  return (
+    <Formik
+      initialValues={user || initialValuesCreateUser}
+      validationSchema={validationSchemaCreateUser}
```

Comment:
make a static class for users validation schema and define the function of validation schema there
src->utils->helpers->ValidationHelpers->UserValidationSchemaHelper